src/apps/insights/utils/supabaseAdapter.ts

```
@@ -0,0 +1,121 @@
+                }
+                
+                const response = { data, error: null };
+                return onResolve ? onResolve(response) : response;
+              } catch (error: any) {
+                const response = { data: null, error };
+                return onReject ? onReject(response) : response;
+              }
+            },
+          }),
+          // For queries without eq filter
+          then: async (onResolve?: any, onReject?: any) => {
```

Biome (2.1.2)

[error] Do not add then to an object.



(lint/suspicious/noThenProperty)